docs/modules/concepts/pages/mail/mime/james.adoc

```diff
@@ -0,0 +1,13 @@
+= MIME James-Specific Description
+:navtitle: James MIME
+
+While the xref:mail/mime/general.adoc[general description] of MIME is
+not sufficient for building a complex system like James, the
+xref:mail/mime/technical.adoc[technical specifications] are unfortunately
+very messy and overly-complex due to their history and the context in which
+they were developed.
+
+To make dealing with emails possible and practical, James has defined its own
+version of what it means to be an email.
```

Comment:
Could somebody kindly tell me what this is?

How **does** James define a MIME Message?

The subject was touched on by @chibenwa [in this thread](https://www.mail-archive.com/server-dev@james.apache.org/msg65869.html).

Reply:
I'm not sure it's so important in an introduction. For me it's meaningful only for customizing James.

Reply:
Point taken... but I feel that I still need to understand. :-)

Reply:
This code is now outdated, so I will resolve the comment.